src/families/ethereum/modules/compound.js

```diff
@@ -471,23 +478,35 @@ export async function digestTokenAccounts(
         // cOUT => REDEEM
         const rates = await fetchHistoricalRates(
           ctoken,
-          ctokenAccount.operations.map((op) => op.date)
+          ctokenAccount.operations.reduce(
+            (prev, op) => (op.blockHeight ? [...prev, op.blockHeight] : prev),
```

Comment:
- maybe reduce+spread is not helping perf, could do more specific logic maybe?
- would we want to dedup even tho there are low chance to have two ops on same block?